src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ILanguageServerExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using OmniSharp.Extensions.LanguageServer.Server;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    public static class ILanguageServerExtensions
+    {
+        public static Task InitializedAsync(this ILanguageServer languageServer, CancellationToken cancellationToken)
+        {
+            var assembly = Assembly.Load("OmniSharp.Extensions.LanguageServer");
```

Comment:
Why do we need to do `Assembly.Load` when you're using `OmniSharp.Extensions.LanguageServer.Server` above?

Reply:
Ya super fair point, in the inproc PR I actually change this to use `typeof(LanguageServer).GetMethod(...)`